pyproject.toml

```diff
@@ -0,0 +1,128 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyroengine"
+description = "Wildfire detection on edge devices"
+authors = [
+    {name = "Pyronear", email = "contact@pyronear.org"}
+]
+readme = "README.md"
+requires-python = ">=3.6,<4"
+license = {file = "LICENSE"}
+keywords = ["pytorch", "deep learning", "vision", "cnn", "wildfire"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+dynamic = ["version"]
+dependencies = [
+    "Pillow>=8.4.0",
```

Comment:
it's not better to use exact versions for deployment ?

Reply:
Yes it is, but I figured the package should have loose specifiers, and `src/requirements.txt` should have fixed ones :)

Reply:
Actually, let me set poetry in place for the `src`, that will help us

Reply:
will be resolved in #102